lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,28 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def manifest_and_backup_paths
```

Comment:
(eek my previous comment is gone don't know what happened)

DRY isn't always good. There are bad DRY. Also DRY doesn't necessarily improve maintainability and readability.

Considering that capistrano tasks are pretty procedural, it would be better to explicitly call two methods, instead of adding a method to wrap two methods in one.

Also Capistrano has two block methods, `with` and `within_...` to wrap **host** environments. They must be block methods (SSHKit). So adding a block method to just wrap two methods can confuse readers.

I really appreciate your time and efforts on it. Like you, I wanted to keep the code quality of this gem pretty high, so that's why I jumped into your PR.